### ISSUE TITLE
Add No-SQL database connection to deployment and local development setup

### ISSUE THREAD

**Issue by TomRomeo** [kioku-project/kioku#275]
## Description
To be able to develop our new card concept, we decided to offload the card content, notes and templates to a No-SQL database, as this data will be unstructured and highly individual. Before we can start development in the backend that uses this database, we have to add it to our deployment and local development setup.

## Todo
- [ ] Research which No-SQL Database to use
- [ ] Research how to communicate with No-SQL Database
